Makefile

```diff
@@ -53,6 +62,7 @@ LIBS = \
 #FFLAGS = -g -Wall -O0 -fopenmp $(INCLUDES)
 
 CFLAGS = -Wall -O3 -fopenmp $(INCLUDES) -DARCH="Linux" -DSPOOLES -DARPACK -DMATRIXSTORAGE -DUSE_MT
+CFLAGS =  -w  -O3 -fopenmp $(INCLUDES) -DARCH="Linux" -DSPOOLES -DARPACK -DMATRIXSTORAGE -DUSE_MT
```

Comment:
Why changing from `-Wall` to `-w`?
If it was intended to add the `-Wextra`, then let's just add it explicitly.

Or is this a change that comes from the CalculiX v2.21 Makefile?

Same below.